README.md

```diff
@@ -5,29 +5,48 @@ You can get started right away with [Microk8s](https://microk8s.io/) - the might
 You can install MicroK8s on your Ubuntu laptop, workstation, or nodes in your workgroup or server cluster with just one command - ```snap install microk8s --classic```. Learn more at [microk8s.io](https://microk8s.io/).
 
 ## Spark-client Snap
-The ***spark-client*** snap includes the scripts [spark-submit](/docs/submit.md), [spark-shell](/docs/shell.md), [pyspark](/docs/pyspark.md) and other tools for managing ***Apache Spark*** jobs for ***Kubernetes***.
+The ***spark-client*** snap includes the scripts to run jobs via [spark-submit](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953), 
+or using [interactive shells](https://discourse.charmhub.io/t/spark-client-snap-tutorial-interactive-mode/8954), and other tools for 
+managing ***Apache Spark*** jobs for ***Kubernetes***.
 
 ## Setup
```

Comment:
Maybe we miss a prerequisites section here. What do you need to get started with this? You need one or more Ubuntu LTS 22.04 servers or workstations, configured to run MicroK8s, with an S3 compatible object storage system like Minio or Canonical Charmed Ceph (with Rados Gateway)